reactioncommerce/reaction
# Fix: Undefined property: Reaction.Router.current().queryParam

Filtering via URL query param did not work any more. This is because the Router's current route has no property called `queryParams`. Correct would be `query`.

Closes #3496 

## To test
- Create one product between 10 and 20 $
- Create a second product over 20 $
- paste this URL into address bar of browser: http://localhost:3000/?price.min=10&price.max=20
- only the product with a price lower than $20 should be displayed, not both.
